src/mutations/placeOrder.test.js

```diff
@@ -155,7 +155,7 @@ test("places an anonymous $0 order with no cartId and no payments", async () =>
         },
         shopId: orderInput.shopId,
         totalItemQuantity: 1,
-        type: "shipping",
```

Comment:
Currently we are not testing the functionality of creating multiple fulfillment groups here as it should be tested in the addCartItems mutations.
Hence we are changing the expected 'type' of group to the default 'mockType' in the test file.